resources/js/components/families/FamilyTree.vue

```diff
@@ -519,6 +541,47 @@ export default {
                 this.closeModal();
             });
         },
+
+        addFounder() {
+            let founder_id = $(this.founderField).val();
+            let entity_id = $(this.entityField).val();
+
+            if (!founder_id || !entity_id) {
+                // Nothing, ignore
+                this.closeModal();
+                return;
+            }
+
+            let url2 = this.entity_api.replace('/0', '/' + entity_id);
+            axios.get(url2).then((res) => {
+                let entity = res.data;
+                this.addFounderEntity(entity);
+                this.isDirty = true;
+            });
+            
+            let url = this.entity_api.replace('/0', '/' + founder_id);
```

Comment:
It's late so my brain might be dead, but why do the 2 requests in sync and not after one another by putting the second one inside the then() of the first one?